htdocs/ticket/class/ticket.class.php

```diff
@@ -500,6 +500,14 @@ public function create($user, $notrigger = 0)
 				if ($this->add_contact($this->fk_user_assign, 'SUPPORTTEC', 'internal') < 0) {
 					$error++;
 				}
+				if (!$error && !$notrigger) {
+					// Call trigger
+					$result = $this->call_trigger('TICKET_ASSIGNED', $user);
```

Comment:
Trigger for creaate action must be called at end of code. Also actions must be CRUD action in priority, when possible, here the trigger CREATE seems already called few lines after making a duplicate trigger call.